### PR TITLE
Update ray example to reflect Hydra 1.1 changes

### LINKED ISSUE #1471: [Bug] Ray plugin ConfigStore schema need to be renamed
# 🐛 Bug
## Description
The Ray plugin (and possibly the other plugins as well) have not been fully migrated to Hydra 1.1. In particular, executing a command like `python my_app.py -m hydra/launcher=ray_aws` with a custom `ray_aws.yaml` config (as described in the plugin documentation) raises a deprecation warning. The fix may be as simple as just changing the schema names [here](https://github.com/facebookresearch/hydra/blob/master/plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_config.py#L257) as suggested in the Hydra 1.0 --> 1.1 migration docs. It's not a critical issue, but it will need to be resolved before Hydra 1.2 is released.

## Checklist
- [ x] I checked on the latest version of Hydra
- [ x] I created a minimal repro (See [this](https://stackoverflow.com/help/minimal-reproducible-example) for tips).

## To reproduce

I've created a minimal example here. 

https://github.com/samuelstanton/hydra-ray-demo

** Stack trace/error message **
```
UserWarning: 
'hydra/launcher/ray_aws' is validated against ConfigStore schema with the same name.
This behavior is deprecated in Hydra 1.1 and will be removed in Hydra 1.2.
See https://hydra.cc/docs/next/upgrades/1.0_to_1.1/automatic_schema_matching for migration instructions.
```

## Expected Behavior
The example should run without warnings.

## System information
- **Hydra Version** :  1.1.0dev5
- **Python version** : 3.8.8
- **Virtual environment type and version** : conda 4.9.2
- **Operating system** : macOS Catalina, 10.15.7

### PR DESCRIPTION
**Pull request by jieru-hu**
Closes #1471 
Verified the example apps run as expected.

### CODE REVIEW

File: website/docs/plugins/ray_launcher.md [hunk 1]
```diff
@@ -144,12 +144,59 @@ $ python my_app.py --multirun task=1,2,3
 </details>
 
 
-<details><summary>Upload & Download from remote cluster</summary>
+##### Overriding default `ray_aws` config
```
Comment (omry) [inline website/docs/plugins/ray_launcher.md:147]: I think we need generic explanation page to explain how to configure plugins in general in Hydra 1.1 (that all plugins can refer to from their docs).
It doesn't make sense for each plugin doc page to have to explain the same thing.

What do you think?

**Comment by omry** [comment]
> #1471
> Verified the example apps run as expected.

You want `Fixes #1471` or `Closes #1471`.

File: website/docs/plugins/ray_launcher.md (hunk 1 above)
Comment (jieru-hu) [reply website/docs/plugins/ray_launcher.md:147]: Makes sense. I will create a new page for configuring plugins.

**Comment by jieru-hu** [comment]
> > #1471
> > Verified the example apps run as expected.
> 
> You want `Fixes #1471` or `Closes #1471`.

updated the  description :)

**Comment by jieru-hu** [comment]
I've created https://github.com/facebookresearch/hydra/pull/1506 to address the comment, once that one lands, I will fix this PR accordingly.

**Comment by omry** [comment]
sounds good.

**Comment by jieru-hu** [comment]
with #1506 merged, it makes sense for this PR to only modify the example apps

for doc changes, all plugin doc will be updated in https://github.com/facebookresearch/hydra/issues/1582

**Comment by omry** [comment]
> with #1506 merged, it makes sense for this PR to only modify the example apps
> 
> for doc changes, all plugin doc will be updated in #1582

Since it looks like for each plugin, we will want to update both the example and the documentation, it could be easier to complete the Ray changes in this PR, including the docs.

**Review by omry — APPROVED** [review]
looking good.
consider finishing the doc changes for this plugin.

**Comment by jieru-hu** [comment]
> looking good.
> consider finishing the doc changes for this plugin.

will do!

**Comment by omry** [comment]
> > looking good.
> > consider finishing the doc changes for this plugin.
> 
> will do!

I meant in this PR :).
never mind, a different PR is fine.

**Pull request merged**